src/symbol-icon-theme.json

```diff
@@ -1280,6 +1285,7 @@
 		".vscode": "folder-blue",
 		".next": "folder-gray",
 		".turbo": "folder-red",
+		".nuxt": "folder-green",
```

Comment:
Maybe create a folder with the small nuxt icon as variant could be better

Reply:
Like Github folder for example

Reply:
I like the idea, I'll keep it in mind.

Reply:
Look at PR #181

Reply:
@FabianMontoya I was testing and it really doesn't look very good, the icon looks too small